mypy/checker.py

```diff
@@ -3954,6 +3954,81 @@ def conditional_callable_type_map(self, expr: Expression,
 
         return None, {}
 
+    def find_type_equals_check(self, node: ComparisonExpr, expr_indices: List[int]
+                               ) -> Tuple[TypeMap, TypeMap]:
+        """Narrow types based on any checks of the type ``type(x) == T``
+
+        :param node: The node that might contain the comparison
+
+        :param expr_indices: The list of indices of expressions in ``node`` that are being compared
```

Comment:
Style nit: We generally use this style for documenting arguments:

```
Args:
    node: The node that might ...
    expr_indiced: The list of indices ...
```

This is a minor thing, however, and this doesn't block merging this PR.

Reply:
I made this change in #10378 because I couldn't figure out how to add more commits to this branch after it's already been merged.